pkg/capi/edit/cluster.x-k8s.io.cluster/ClusterConfig.vue

```diff
@@ -307,7 +304,7 @@ export default (Vue as VueConstructor<
       });
     },
     clickedType(obj: {[key:string]: any}) {
-      this.clusterClassObj = this.clusterClasses.find((x: ClusterClass) => x.metadata.name === obj.id);
+      this.clusterClassObj = this.clusterClasses.find((x: ClusterClass) => x.metadata.uid === obj.id);
```

Comment:
When I try to run `yarn build-pkg capi` this line throws an error because `uid` is not defined in the `ClusterClass` interface.

Is there a reason to rely on `metadata.uid` instead of `id` here? The latter would be more consistent with dashboard code generally